build-data/generated/tracking_parameters.json

```diff
@@ -0,0 +1,1135 @@
+                "www.t-online.de/unterhaltung/tv/id_17600248/dana-vavrova-gestorben-sie-ist-bekannt-aus-herbstmilch-und-die-gustloff-.html",
+                "www.pcworld.com",
+                "www.tvguide.com",
+                "autotrader.com",
+                "www.edmunds.com/used-toyota-prius-c-tampa-fl"
+            ],
+            "requests3p": {
+                "prevalence": 0.096,
+                "entities": {
+                    "Google LLC": 0.209,
+                    "id.a-mx.com": 0.055,
+                    "Sailthru, Inc": 0.047,
+                    "Adobe Inc.": 0.047,
+                    "ANIVIEW LTD": 0.036,
+                    "StatCounter": 0.035,
+                    "Akamai Technologies": 0.021,
+                    "Index Exchange, Inc.": 0.021,
+                    "Salesforce.com, Inc.": 0.018
+                },
+                "domains": {
+                    "doubleclick.net": 0.086,
+                    "google.com": 0.075,
+                    "a-mx.com": 0.055,
+                    "sail-personalize.com": 0.047,
+                    "aniview.com": 0.036,
+                    "statcounter.com": 0.035,
+                    "googleadservices.com": 0.029,
+                    "omtrdc.net": 0.028,
+                    "akamaihd.net": 0.021
+                }
+            },
+            "cookies": {
+                "prevalence": 0.001,
+                "firstParty": 301,
+                "thirdParty": 15,
+                "entities": {
+                    "iSpot.tv": 0.267,
+                    "Unbounce": 0.267,
+                    "www.directstoves.com": 0.133,
+                    "www.druni.eslax": 0.133,
+                    "ZOHO Corporation": 0.067,
+                    "zoho.com.au": 0.067,
+                    "www.novantmychart.org": 0.067
+                },
+                "domains": {
+                    "www.ispot.tv": 0.267,
+                    "bf098b5490a14f9d86e99cc566def128.pages.ubembed.com": 0.267,
+                    "www.directstoves.com": 0.133,
+                    "www.druni.eslax": 0.133,
+                    "zoho.in": 0.067,
+                    "zoho.com.au": 0.067,
+                    "www.novantmychart.org": 0.067
+                }
+            }
+        },
+        "fb_ref": {
+            "prevalence": 0.096,
+            "exampleSites": [
+                "itsfoss.com/use-windows-applications-linux",
+                "news.google.fr",
+                "www.businessinsider.com",
+                "www.thespruce.com",
+                "www.testberichte.de/test/produkt_tests_triangle_ventis_xs_p14548.html",
+                "account.mail.ru",
+                "www.rottentomatoes.com/tv/the_wheel_of_time",
+                "stacks.cdc.gov",
+                "pay.amazon.com",
+                "news.google.de"
+            ],
+            "requests3p": {
+                "prevalence": 0.095,
+                "entities": {
+                    "Google LLC": 0.205,
```

Comment:
kinda weird that we see facebook collecting `fbclid` but not any of those other `fb_*` params - just an observation